rancher/context.go

```diff
@@ -166,14 +167,24 @@ func (c *Context) sanitizedProjectName() string {
 	return projectName
 }
 
-func (c *Context) loadClient() (*rancherClient.RancherClient, error) {
+func (c *Context) loadClient() (*client.RancherClient, error) {
 	if c.Client == nil {
 		if c.Url == "" {
 			return nil, fmt.Errorf("RANCHER_URL is not set")
 		}
 
-		if client, err := rancherClient.NewRancherClient(&rancherClient.ClientOpts{
-			Url:       c.Url,
+		url, err := url.Parse(c.Url)
+		if err != nil {
+			return nil, err
+		}
+
+		base := path.Base(url.Path)
```

Comment:
If the URL does not already end in `/v2-beta` or `/schemas`, append `/v2-beta` to the end.